src/package-request.js

```diff
@@ -107,6 +107,16 @@ export default class PackageRequest {
       // times
       data = Object.assign({}, data);
 
+      // local dependencies are not required in having a version number so they get
+      // the current timestamp to require it
+      if (!data.version) {
+        data.version = Date.now().toString() + '.0.0';
```

Comment:
`Date.now()` would be a no-go because it would make installs behave differently from one to the other (because the versions would be different), but just `0.0.0` would be ok 🙂

Reply:
`0.0.0` was also my first choice but thought this could end up in caching issues?

Reply:
Ok changed to 0.0.0